### PR TITLE
*: support write slow query log into seperate files

### PR DESCRIPTION
**Pull request by tiancaiamao**
Support this config option, then slow query log will be redirect to the separate files:

```
[log]
slow-query-file = "slow-query.log"
```

This PR also remove `pd/pkg/logutil` and create a `tidb/util/logutil` package, because log config in Pd is somehow different with TiDB. For example, TiDB doesn't need to handle etcd log, while Pd doesn't need to collect slow query log.

Using [`slow_query_log` and `slow_query_log_file`](https://dev.mysql.com/doc/refman/5.7/en/slow-query-log.html) SQL variable to control the slow query log is not supported deliberately, because global session variable will change all TiDB instance's behavior, the deploy environment of the instances maybe diverse, however.

### CODE REVIEW

**Comment by shenli** [comment]
Please fix CI.

File: config/config.toml.example [hunk 1]
```diff
@@ -37,6 +37,9 @@ format = "text"
 # Disable automatic timestamps in output
 disable-timestamp = false
 
+# Stores slow query log into seperate files.
+#slow-query-file = ""
```
Comment (zimulala) [inline config/config.toml.example]: Remove `#`?

**Comment by zimulala** [comment]
Add a unit test for `SlowQueryLogger` ?

File: config/config.toml.example (hunk 1 above)
Comment (tiancaiamao) [reply config/config.toml.example]: We should not set it by default, merge the output of slow query to standard log will be helpful for debugging.

**Comment by winkyao** [comment]
Why you move the `logutil/log.go` from vendor/pd to TiDB, not just copy the file?

**Comment by tiancaiamao** [comment]
> This PR also remove pd/pkg/logutil and create a tidb/util/logutil package, because log config in Pd is somehow different with TiDB. For example, TiDB doesn't need to handle etcd log, while Pd doesn't need to collect slow query log.

@winkyao

**Comment by coocood** [comment]
LGTM

**Pull request merged**